src/EventHub/EventHub/Utilities/EventHubsClient.cs

```diff
@@ -66,7 +66,7 @@ public IEnumerable<PSNamespaceAttributes> ListNamespacesBySubscription()
             return resourceList;
         }
 
-        public PSNamespaceAttributes BeginCreateNamespace(string resourceGroupName, string namespaceName, string location, string skuName, int? skuCapacity, Dictionary<string, string> tags, bool? isAutoInflateEnabled, int? maximumThroughputUnits, bool? isKafkaEnabled)
+        public PSNamespaceAttributes BeginCreateNamespace(string resourceGroupName, string namespaceName, string location, string skuName, int? skuCapacity, Dictionary<string, string> tags, bool? isAutoInflateEnabled, int? maximumThroughputUnits, bool? isKafkaEnabled, NamespaceState? state = NamespaceState.Active)
         {
```

Comment:
Is state useful?

Reply:
no, we need to remove the parameter, I will mark its as deprecated as mentioned in Breaking-change.md

Reply:
marked as breaking change - deprecated